cg/meta/deliver.py

```diff
@@ -210,6 +213,21 @@ def deliver_sample_files(
             f"There were {number_previously_linked_files} previously linked files and {number_linked_files_now} were linked for sample {sample_id}, case {case_id}"
         )
 
+        if self.delivery_type == Workflow.MICROSALT:
```

Comment:
Will this check work as expected or will the `delivery_type` just be fastq?

Reply:
hmm, actually I think you're right. This will not work. The type will be FASTQ

Reply:
I sent down the case and validate that the `data_analysis` set on it is `microsalt`.

Reply:
Yeah, if I don't misremember thats the same logic we have in the other CLI command that does it 👍